docs/guides/user/cli-client.md

```diff
@@ -51,13 +51,15 @@ Options:
 Commands:
   analyze         Analyze timelines.
   config          Configuration for this CLI tool.
-  events          Manage events
+  events          Manage event.
   import          Import timeline.
+  intelligence    Manage intelligence within a sketch.
+  sample-data     Commands for generating sample or test data.
   saved-searches  Managed saved searches.
-  search          Search and explore.
+  search          Searches and explores events within a Timesketch sketch.
+  sigma           Manage sigma rules.
```

Comment:
Do you want to add the sigma stuff here when we are actually in the process of removing it from Timesketch?

Reply:
I just wanted to have the doc page to match reality, once we remove it, we will remove from the doc as well.